hypothesis-python/tests/nocover/test_health_checks.py

```diff
@@ -107,21 +107,20 @@ def test(n):
             test()
 
 
-def test_does_not_trigger_health_check_when_most_examples_are_small(monkeypatch):
-    with deterministic_PRNG():
```

Comment:
I'm worried about nondeterministic failures here; maybe we can apply `@seed(idx)` instead with the loop variable?

Reply:
done

I don't care too much about nondeterministic failures honestly, I'd actually prefer to use less seeding in our tests, so that we don't get silently passing seeded tests when they would have caught a real failure eventually. Of course the flipside is dealing with flakes.